server/models/errorsModel.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
```

Comment:
change to: `const { Schema, connect, model } = require('mongoose')`, convert elsewhere in file as needed, and double check it works.